app/controllers/hubstats/deploys_controller.rb

```diff
@@ -47,15 +51,35 @@ def find_net_additions(deploy_id)
       return total_additions - total_deletions
     end
 
+    # finds the total number of additions for all pull requests in this deploy
+    def find_additions(deploy_id)
+      deploy = Hubstats::Deploy.find(deploy_id)
+      pull_requests = deploy.pull_requests
+      total_additions = 0
+      pull_requests.each do |pull|
+        total_additions += pull.additions.to_i
+      end
+      return total_additions
+    end
+
+    # finds the total number of deletions for all pull requests in this deploy
+    def find_deletions(deploy_id)
+      deploy = Hubstats::Deploy.find(deploy_id)
+      pull_requests = deploy.pull_requests
+      total_deletions = 0
+      pull_requests.each do |pull|
+        total_deletions += pull.deletions.to_i
+      end
+      return total_deletions
+    end
+
     # returns the total amount of comments from all pull requests in a deploy
     def find_comment_count(deploy_id)
       deploy = Hubstats::Deploy.find(deploy_id)
       pull_requests = deploy.pull_requests
       total_comments = 0
       pull_requests.each do |pull|
-        if pull.comments
-          total_comments += Hubstats::Comment.belonging_to_pull_request(pull.id).includes(:user).created_since(@timespan).count(:all)
-        end
+          total_comments += Hubstats::Comment.belonging_to_pull_request(pull.id).created_since(@timespan).count(:all)
```

Comment:
It doesn't matter whether a pull has comments or not. The number of comments a pull says it has may differ from the comments that actually belong to this pull.